src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const heart = props.liked ? '❤️' : '🤍';
+  const handleLike = () => {
+    props.updateLike(props.id);
+  };
+  const senderClass =
+    props.sender === 'Vladimir' ? 'chat-entry local' : 'chat-entry remote';
```

Comment:
NIce way to figure out whether to apply the local or remote css class. But what if the chat participants were other senders? Could we determine the "local sender" higher up and then pass that information down so that we can avoid hard-coding a name?